mmsmessage/mms_message.go

```diff
@@ -1,10 +1,13 @@
-package messagebird
+package mmsmessage
```

Comment:
What do you think about naming it just `mms`?

Reply:
Totally agree!